extension/js/common/api/key-server/attester.ts

```diff
@@ -53,7 +39,15 @@ export class Attester extends Api {
    * the actual api accepts either email, fingerprint or longid
    */
   public lookupFingerprint = async (fingerprintOrLongid: string) => {
-    return await this.lookupEmail(fingerprintOrLongid);
+    if (fingerprintOrLongid.includes('@')) {
+      return await this.lookupEmail(fingerprintOrLongid);
+    }
```

Comment:
Actually I think this should be a `throw new Error`, since we already have distinct method for email and fingerprint/longid. If that ever blows up, then we are calling this wrongly, which would be good to know :-)